### ISSUE TITLE
Steal should come pre-packaged with meta and map configurations for popular packages.

### ISSUE THREAD

**Issue by mjstahl** [stealjs/steal#1395]
Every time I start a new project running steal and I want to use Chai or Mocha (or many other popular packages that are escaping me right now) I go look up how to set up the `meta` and `map` properties in steal's configuration.

I believe packages that are used a lot (we can judge that by looking at projects that Bitovi is currently developing) we should include `meta` and `map` properties pre-configured.

**Comment by justinbmeyer** [comment]
@matthewp can one package be configured from another package?

@mjstahl instead of bundling it with steal, and having to support changes to these other packages (and possibly handling different versions) ((and having to change steal's version number if things break)) I think we should either:

- Create `steal-chai`, `steal-mocha` packages like the `steal-qunit` package.
- Support "configuration" packages.  You could make the configuration in these packages.

**Comment by justinbmeyer** [comment]
Actually, there is a `steal-mocha`: https://github.com/stealjs/steal-mocha

**Comment by matthewp** [comment]
So what if we had a `steal-popular-packages` npm package that a dev just had to install that then autoconfigured? That would avoid the needing to include that stuff in steal but also make it super easy to use. Would that be an 85% solution?

**Comment by mjstahl** [comment]
I think that is a good compromise.

**Comment by matthewp** [comment]
# Proposal

__tldr;__ Provide a new package that has preconfigurations for a variety of popular npm packages.

## Problem

As stated in this issue, certain packages always need configuration, like jQuery widgets in Bootstrap, or those that need to use a global build like Chai. Having to remember to configure these in each of your projects can be tedious.

## Solution

Create a new package, perhaps `steal-preconfigured` that includes configurations for a variety of popular npm packages. Users can contribute to this project by adding their own configurations.

Secondarily, improve the documentation around popular packages on stealjs.com. In particular, point to existing Steal packages that handle the integration, such as steal-qunit for QUnit, steal-socket.io for Socket.IO, etc.

**Comment by eben-roux** [comment]
Is it not perhaps another option to have `steal-tools` provide a command line interface to perform the configuration for you?

`steal-tools -configure jquery`
